src/test/scala/edu/berkeley/ce/rockslicing/SeedJointSelectorSpec.scala

```diff
@@ -0,0 +1,212 @@
+package edu.berkeley.ce.rockslicing
+
+import org.scalatest.FunSuite
+
+class SeedJointSelectorSpec extends FunSuite {
+
+  private def compareJoints(joint1: Joint, joint2: Joint): Boolean = {
```

Comment:
You should name this something different to indicate that it's an equality check and not a comparison.